parquet-hadoop/src/main/java/org/apache/parquet/hadoop/codec/NonBlockedCompressor.java

```diff
@@ -30,6 +30,8 @@
  */
 public abstract class NonBlockedCompressor implements Compressor {
 
+  private static final int INITIAL_INPUT_BUFFER_SIZE = 4096;
```

Comment:
Is it possible to reuse the config which is defined for both LZ4_RAW and SNAPPY?  https://github.com/apache/parquet-mr/blob/92ac5bc9ecb84a48b68dc35bf7f2b36cb0e6583b/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/codec/SnappyCodec.java#L42

Reply:
Yes, I just updated the patch to use this config.